packages/backend/src/factcollectors/store/branchcountextractorsstore.ts

```diff
@@ -29,29 +45,47 @@ export class BranchCountExtractorsStore implements ExtractorsStore {
     });
   }
 
-  #parseExtractorSchemaTypesFromConfig(
-    config: Config,
-  ): Record<string, ExtractorConfig> {
+  /**
+   * Parses extractor schema types from the given configuration
+   * @param {Config} config - The configuration to parse
+   * @return {Record<string, ExtractorConfig>} - The parsed extractor configurations
+   */
+  #parseExtractorSchemaTypesFromConfig(config: Config) {
+    // Retrieve the 'soundcheck.collectors.branch' configuration from the app-config.yaml file if it exists
     const collectors = config.getOptional('soundcheck.collectors.branch');
 
     // branch collection not configured, so nothing to do.
-    if (!collectors) return {};
+    if (!collectors) {
+      this.#logger.error(`No branch collectors configured, make sure you have a 'soundcheck.collectors.branch' section in your app-config.yaml file.`);
+      return;
+    }
 
     // Zod verification.
     const schema = BranchCountFactCollectorSchema.safeParse(collectors);
```

Comment:
This shoudl be changed to:
`const schema = BranchesDetailsFactCollectorSchema.safeParse(collectors);`

Reply:
I update it back to BranchCount which was the original nomenclature used